Server/BackEndMonitoringServer/include/CJSONFormatterProcess.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#include "CJSONFormatter.h"
+#include "CProcess.h"
```

Comment:
Use forward class declaration instead of including CProcess.h

Reply:
Done!